### DISCUSSION TITLE
Cannot add values in database

### DISCUSSION THREAD

**Discussion by Anjumvj** [learndevopsonline/learndevopsonline#1546] [Q&A]
Hi Raghu,
I have connected Frontend , Backend and Mysql. Then when i go back to frontend to add values to the DB. Nginx status shows error. Cannot proceed. Please find the attachment.
![Screenshot 2023-10-29 131358](https://github.com/learndevopsonline/learndevopsonline/assets/130336724/a76f3c5c-3406-4aff-8154-da0eeb600ffa)

**Comment by sreedharm07** [comment]
remove that.
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/1451d76a-91ac-48e2-b46b-67b709e013f9)

**Reply by Anjumvj** [reply]
![Screenshot 2023-10-29 132507](https://github.com/learndevopsonline/learndevopsonline/assets/130336724/c43f4cd3-37f0-464e-a64b-1564d7910017)
how to proceed then without giving :WQ

**Reply by sreedharm07** [reply]
first enter esc then give :wq

**Reply by Anjumvj** [reply]
Thanks .
